README.md

```diff
@@ -86,6 +86,7 @@ use cases.
 |  9 | Near Protocol                   |    Octopus    |    :100:     |                   [`near-ibc`][near-ibc]                    | :hourglass_flowing_sand: |
 | 10 | Tendermint Light Client         | Electron Labs |   `ics07`    | [`Electron-Labs/tendermint-rs`][electronlabs-tendermint-rs] | :hourglass_flowing_sand: |
 | 11 | Starknet                        |   Informal    |    :eyes:    |                           :eyes:                            |          :eyes:          |
+| 12 | SP1 Tendermint Light Client     |  Interchain   |   `ics07`    |        [`cosmos/sp1-ics07-tendermint`][sp1-tendermint]      | :hourglass_flowing_sand: |
```

Comment:
Since I see the project is a PoC, should we use :hammer_and_pick: here? Is that employed in any devnet/testnet project?

Reply:
I did push it on some testnets, but it is still WIP. Actually, we did decide to commit to it for now, and have partners launch with it if we can have the rest of IBC in Solidity.